chain/chain/src/flat_storage_resharder.rs

```diff
@@ -295,7 +304,9 @@ impl FlatStorageResharder {
                 return FlatStorageReshardingTaskStatus::Failed;
             }
 
-            // TODO(Trisfald): metrics and logs
+            let time = timer.elapsed();
+            batches_done += 1;
+            debug!(target: "resharding", ?time, ?batches_done, "flat storage resharding split shard batch processed");
```

Comment:
Why an event rather than another span?

Reply:
Do you mean span for the loop (or the whole function) + event for every finished iteration? Otherwise I don't understand why the span may be needed for this line.

Reply:
Switched to use a span